topics/single-cell/tutorials/scrna-data-formats/slides.html

```diff
@@ -0,0 +1,578 @@
+
+]
+
+???
+
+Single-cell data consists of 4 main components:
+- gene expression data 
+- metadata about each cell
+- metadata about each gene
+- unstructured metadata about the data collected
+
+All complex single-cell data formats will contain these 4 main sections, it's important to understand what these components are, how to identify them, and their importance in single-cell analysis
+
+---
+
+### Gene expression matrix (Matrix)
+
+.pull-left[
+
+- 2D matrix of expression values
+  - Rows represent genes
+  - Columns represent cells
+
+- Can be in a full or sparse matrix form
+
+]
+
+.pull-right[
+
+![Basic structure of single-cell data with the matrix section highlighted](../../images/scrna-data-formats/gene-expression-diagram-matrix.png)
+
+]
+
+???
+
+The core component of single-cell data is the gene expression matrix. This is a 2D matrix representing the gene expression values for each gene in the sequenced cells.
+
+Typically each row of the matrix represents the genes and the columns represent the cells. However, it is important to note that this is not always the case and some matrices may need to be transposed before further processing.
+
+The matrix can come in two different forms: full or sparse. 
+
+Full matrices act as one large table where every entry contains a value, including genes that were not sequenced (in which case their value will be 0). This is the simplest format but can be inefficient to store as many of the values in the matrix are redundant. 
+
+A more compact method to store these values is with a sparse matrix. This is a compressed representation of the matrix that removes all values that contain no useful information whilst keeping track of the overall structure of the matrix.
+
+---
+
+### Cell metadata (Barcodes)
+
+
+.pull-left[
+
+- Cell identifiers/barcodes
+
+- Patient/sample metadata
+
+- Quality control metrics
+
+<br/><br/>
+
+- **Example barcodes:**
+  - AAACCTGAGTCT
+  - TGCATTAGGGA
+  - CCGTTGACCTAC-0
+  - GATCGATCGTCA-0
+
+]
+
+.pull-right[
+
+![Basic structure of single-cell data with the cells section highlighted](../../images/scrna-data-formats/gene-expression-diagram-cells.png)
+
+]
+
+???
+
+The first section of metadata that will be explored is the cell metadata, commonly referred to as barcodes when stored in a file. This contains metadata on each cell. This metadata includes the cell identifiers/barcodes, data on the origin of each cell, and quality control metrics generated with further analysis tools.
+
+Cell barcodes consist of the 4 nucleotide letters and occasionally suffixed with a dash and a number (which has various meanings). If a file contains a column containing data in this format, then you are likely looking at cell metadata!
+
+---
+
+### Gene metadata (Features)
+
+.pull-left[
+
+- Gene identifiers/Ensembl IDs
+
+- Expression metrics
+
+- Quality control metrics
+
+<br/><br/>
+
+- **Example gene names:**
+  - TP53, BRCA1

```

Comment:
Is there a reason why the gene names are in pairs? If not, then I would separate them. 
It would be good to have some examples of Ensembl IDs here - maybe give pairs of gene names & IDs?

Reply:
Just realised they still are apart of the slides.. oops. Either way I'll make them one column and add in Ensembl IDs